src/bluesearch/k8s/create_indices.py

```diff
@@ -47,12 +64,12 @@
 
 
 def add_index(
+    client: Elasticsearch,
     index: str,
     settings: dict[str, Any] | None = None,
     mappings: dict[str, Any] | None = None,
 ) -> None:
-    client = connect()
-
+    """Add the index to ES."""
```

Comment:
Can we add the description of the parameters and the returned values ?

Reply:
done